src/promptflow/setup.py

```diff
@@ -69,13 +69,7 @@
         ],
     },
     packages=find_packages(),
-    scripts=["pf", "pf.bat"],
-    entry_points={
-        "console_scripts": [
-            "pfazure = promptflow.azure._cli.entry:main",
-            "pfs = promptflow._sdk._service.entry:main",
-        ],
-    },
+    scripts=["pf.bat"],
```

Comment:
may need to take a look at this in next PR

Reply:
yes, I had a discussion with ying and she may take some actions on packaging and upgrading issue along with me.